renderer/src/components/Onboarding.tsx

```diff
@@ -0,0 +1,108 @@
+import { FC, useState } from 'react'
+import { ReactComponent as Back } from './../assets/img/arrow-left.svg'
+import { ReactComponent as Page } from './../assets/img/paginator-page.svg'
+import { ReactComponent as CurrentPage } from './../assets/img/paginator-current.svg'
+
+interface IFooter {
+  page: number,
+  pages: number,
+  next: () => void,
+  prev: () => void
+}
+
+const Footer: FC<IFooter> = ({ page, pages, next, prev }) => {
+  return (
+    <div className="flex justify-between">
+      <button
+        className="btn-secondary-small "
+        disabled={page === 0}
+        onClick={prev}>
+        <span className="">Back</span>
+      </button>
+      <div className='flex flex-row items-center space-between gap-3'>
+        {[...Array(pages - 1).keys()].map((_, index) => {
+          return index === page ? <CurrentPage key={index}/> : <Page key={index}/>
+        })}
+      </div>
+      <button
+        className="btn-secondary-small "
+        onClick={next}>
+        <span className="">Continue</span>
+      </button>
+    </div>
+  )
+}
+
+interface IOnboarding {
+  onFinish: () => void
+}
+
+const Onboarding: FC<IOnboarding> = ({ onFinish }) => {
+  const [page, setPage] = useState(0)
+  const pages = 3
+  const next = () => {
+    setPage((page + 1) % (pages))
+  }
+
+  const prev = () => setPage(page - 1)
```

Comment:
`next` protects against going over 3, `prev` doesn't. Was this intentional? I think we don't need this protection here inside `next`, and then could also remove the `pages` variable

Reply:
2e891618b82a949e5177ffcb11afbe6523e8d4c4